index.js

```diff
@@ -148,7 +148,9 @@ gulpSass.sync = function sync(options) {
 // Log errors nicely
 //////////////////////////////
 gulpSass.logError = function logError(error) {
-  gutil.log(gutil.colors.red('[' + PLUGIN_NAME + '] ') + error.messageFormatted);
+  var message = new gutil.PluginError('sass', error.messageFormatted).toString();
+  process.stderr.write(message + '\n');
+  this.emit('end');
```

Comment:
i think you can just return a new gutil.PluginError looking at other gulp plugins using cb like  https://github.com/wearefractal/gulp-coffee/blob/master/index.js#L38